### PR TITLE
Add test to build fleet using dapper

### PR DESCRIPTION
**Pull request by dcermak**

### CODE REVIEW

**Review by evrardjp — APPROVED** [review]

File: test_go.py [hunk 1]
```diff
@@ -19,5 +21,18 @@ def test_go_version(container):
...
+    [
+        GitRepositoryBuild(
+            repository_url="https://github.com/mvarlese/fleet",
+        ).to_pytest_param()
+    ],
+    indirect=["host_git_clone"],
+)
+def test_fleet(host, host_git_clone, dapper):
+    dest, _ = host_git_clone
+    host.run_expect([0], f"cd {os.path.join(dest, 'fleet')} && {dapper}")
```
Comment (evrardjp) [inline test_go.py:38]: shouldn't this be dapper make ?

**Review by evrardjp — CHANGES_REQUESTED** [review]
I think this is not doing the right thing, except if I am misunderstanding.

File: test_go.py (hunk 1 above)
Comment (mvarlese) [reply test_go.py:38]: In my local tests, I simply run "make" from the top-level folder in the fleet project indeed.

Comment (dcermak) [reply test_go.py]: It should be either `dapper build` or `dapper ci`.

**Review by evrardjp — APPROVED** [review]
Ok tracked it on the fleet repo, makes sense to me.
You have to follow the path towards the Dockerfile of fleet to understand this. It surely isn't obvious at first glance, and IMO should not be necessary. Anyway, this makes this tool tested, and should make some folks happy, I guess.

**Pull request merged**